components/commands/browser/accelerator_pref_manager.cc

```diff
@@ -29,7 +31,8 @@ constexpr char kDefaultAcceleratorsPrefs[] = "brave.default_accelerators";
 
 AcceleratorPrefManager::Accelerators GetAcceleratorsFromPref(
     const std::string& pref,
-    PrefService* prefs) {
+    PrefService* prefs,
+    const base::flat_set<int> command_ids) {
```

Comment:
`const base::flat_set<int>& command_ids` is what I assume was supposed to be.

Reply:
Sure is!

Reply:
https://github.com/brave/brave-core/pull/21036